app/models/issue_tracker.rb

```diff
@@ -15,19 +15,19 @@ def tracker
         klass = ErrbitPlugin::Registry.issue_trackers[type_tracker] || ErrbitPlugin::NoneIssueTracker
         # TODO: we need to find out a better way to pass those config to the issue tracker
         klass.new(options.merge(
-          github_repo: app.try(:github_repo),
-          bitbucket_repo: app.try(:bitbucket_repo)
+                    github_repo: app.try(:github_repo),
+                    bitbucket_repo: app.try(:bitbucket_repo)
```

Comment:
This cop doesn't make sense to me. The previous format is more legible to my eyes. Is there a way to enforce that format instead of this new one?